### PR TITLE
Canvas: Fix ellipse migration

### PR DESCRIPTION
**Pull request by Develer**
**What is this feature?**

This PR fixes the canvas ellipse migration issue when the value is overridden by an empty value.

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by Develer** [comment]
After a couple of tests, I think it happens after double migration e.g. when the user migrates from 11.0.smth to 11.0.smth2.

**Comment by Develer** [comment]
is this the correct way to check the migration version?
https://github.com/grafana/grafana/blob/a360cf0c8564179261c3071155e6824105a98771/public/app/plugins/panel/canvas/migrations.ts#L21

File: public/app/plugins/panel/canvas/migrations.ts [hunk 1]
```diff
@@ -25,9 +25,15 @@ export const canvasMigrationHandler = (panel: PanelModel): Partial<Options> => {
...
+          }
+          if (element.config.borderColor) {
+            element.border.color = element.config.borderColor;
+          }
+          if (element.config.width) {
+            element.border.width = element.config.width;
+          }
 
           // Remove the ellipse specific background and border config
           delete element.config.backgroundColor;
```
Comment (drew08t) [inline public/app/plugins/panel/canvas/migrations.ts:39]: Should we also throw these inside the if statements?

Comment (Develer) [reply public/app/plugins/panel/canvas/migrations.ts:39]: yeah, I was thinking about it, I'll do

**Review by codeincarnate — APPROVED** [review]
Have it a shot and appears to work correctly 😄

File: public/app/plugins/panel/canvas/migrations.ts (hunk 1 above)
Comment (drew08t) [reply public/app/plugins/panel/canvas/migrations.ts:39]: Might as well. 😄

**Review by baldm0mma — APPROVED** [review]
Nice! I agree with @drew08t, I think throwing the deletes in the conditions is a good idea. Once that's done, looks good!

**Pull request merged**